base/src/main/java/com/streamliners/base/BaseViewModel.kt

```diff
@@ -26,7 +26,7 @@ open class BaseViewModel : ViewModel() {
 
     open fun init() { }
 
-    private fun handleException(throwable: Throwable) {
+     fun handleException(throwable: Throwable) {
```

Comment:
- Also make BaseActivity's similar function public
- minor : extra starting space in L#29